### PR TITLE
Add MM1 continuum parameters, modify lines parameters for MM2 and MM3

### PR DESCRIPTION
**Pull request by nonyt**
Changes on scripts other imaging_parameters should be ignored
Some line (CO, SiO) parameters (deconvolver) were used in older versions, need to be check and removed

### CODE REVIEW

File: reduction/continuum_imaging_selfcal.py [hunk 1]
```diff
@@ -857,7 +857,8 @@ def sethistory(prefix, selfcalpars=None, impars=None, selfcaliter=None):
                  interp="linear", applymode='calonly', calwt=False)
 
 
-    for robust in (0, -2, 2, -1, 1, -0.5, 0.5):
+    #for robust in (0, -2, 2, -1, 1, -0.5, 0.5):
+    for robust in (0, 2):
```
Comment (keflavich) [inline reduction/continuum_imaging_selfcal.py:861]:
```suggestion
    for robust in (0, -2, 2, -1, 1, -0.5, 0.5):
```

File: reduction/imaging_parameters.py [hunk 2]
```diff
@@ -445,6 +445,61 @@
...
+            4: [0, 3, 9, 27],
+            "final": [0, 3, 9, 27, 54],
+        },
+        "maskname": {
+            0: "W43-MM1_B6_dirty_12M.crtf",
+            1: "W43-MM1_B6_early-iter_12M.crtf",
+            2: "W43-MM1_B6_early-iter_12M.crtf",
+            3: "W43-MM1_B6_late-iter_12M.crtf",
+            4: "W43-MM1_B6_last-iter_12M.crtf",
+            "final": "W43-MM1_B6_final_12M.crtf",
```
Comment (keflavich) [inline reduction/imaging_parameters.py:499]: add these CRTF files too

File: reduction/masking_script_casa5.py [hunk 3]
```diff
@@ -1,13 +1,16 @@
...
 
-image_filename = 'G008.67_B6_spw1_12M_sio.image'
-mask_filename = 'G008.67_B6_spw1_12M_sio_2sigma' # Don't include .mask
-mask_threshold = 20 # In mJy
+#viewer('/lustre/roberto/ALMA_IMF/lines/imaging_results/W43-MM2_B6_spw1_12M_sio.residual')
+
+#image_filename = '/lustre/roberto/ALMA_IMF/lines/imaging_results/W43-MM2_B6_spw1_12M_sio.image'
+image_filename = '/lustre/roberto/ALMA_IMF/lines/imaging_results/W43-MM2-old/multiscale/W43-MM2_B6_spw5_12M_12co.image'
+mask_filename = '/lustre/roberto/ALMA_IMF/lines/imaging_results/W43-MM2_B6_spw5_12M_12co_multi_2sigma' # Don't include .mask
+mask_threshold = 3.0 # In mJy sigma_sio_MM2=2.5 12co_MM2=1.5
```
Comment (keflavich) [inline reduction/masking_script_casa5.py:13]: are these changes intentional?  if so, we can keep them

**Review by keflavich — CHANGES_REQUESTED** [review]
Could you add the CRTF files, and remove the changes to continuum_imaging_selfcal?

**Comment by nonyt** [comment]
I added the masks and restored 2 scripts to the original states. Is it fine now or should I do another pull request?

**Pull request merged**